guides/v2.1/config-guide/bootstrap/magento-bootstrap.md

```diff
@@ -40,7 +40,7 @@ To run the Magento application, the following actions are implemented in <a href
 
 <div class="bs-callout bs-callout-info" id="info">
 <span class="glyphicon-class">
-  <p>The assertions that the Magento application is installed and not in maintenance mode is the default behavior of the <code>\Magento\Framework\App\Bootstrap</code> class. You can modify it using an entry point script when creating the bootstrap object.</p></span>
+  <p>The assertions that the Magento application is installed and not in maintenance mode is the default behavior of the <a href="{{ site.mage2000url }}lib/internal/Magento/Framework/App/Bootstrap.php" target="_blank">\Magento\Framework\App\Bootstrap</a> class. You can modify it using an entry point script when creating the bootstrap object.</p></span>
```

Comment:
@RakeshJesadiya You can not add HTML tag in markdown, Please update it.

Reply:
This is ok because it is HTML within an HTML info note (which we need to keep).

Make sure that the directory path, even when linked, shows in code formatting as well.